lib/navigation_bar_controller.dart

```diff
@@ -28,7 +29,6 @@ class _AppBottomNavBarControllerState extends State<AppBottomNavBarController> {
   List<Widget> pages;
   PageController _pageController;
   int _selectedIndex;
-  bool profileSelected = false;
```

Comment:
This is no longer needed, because clicking the Settings Icon opens a new screen instead of a different page on the same screen